cmd/thanos/store.go

```diff
@@ -102,7 +102,7 @@ func runStore(
 	dataDir string,
 	grpcAddr string,
 	httpAddr string,
-	gossipTimestampsFn func(mint int64, maxt int64),
+	peer *cluster.Peer,
```

Comment:
Do we need to pass whole structure? just SetTimestamps interface should fine?

Reply:
or the function we had previously

Reply:
We are printing the peer name everywhere as well now. Once we start injecting several things, we might as well just pass the full thing if it's just the scope of a single function anyway.